include/oneapi/ccl/config.h

```diff
@@ -1,43 +0,0 @@
-#pragma once
```

Comment:
I think I need to check when and why it disappeared

Reply:
@Maria1Petrova this is an auto-generated build file; it should regenerate on each build; the issue stems from me using separate machine for packing and another one for testing instead of packing the post-test content.

This file is not versioned in inner-source and I think it would actually be better not to include this auto-generated file in the opensource. @nikitaxgusev do you agree ?